apps/baseline/models.py

```diff
@@ -227,7 +227,14 @@ class Language(models.TextChoices):
         verbose_name=_("Population Estimate"),
         help_text=_("The estimated population of the Livelihood Zone during the reference year"),
     )
-
+    currency = models.ForeignKey(
+        Currency,
+        default="USD",
```

Comment:
Ideally we don't want a default on the model. I think if you set the field as mandatory and don't give a default, then when you generate the migration Django will ask you to specify the default - which only applies during the migration.